R/expect-setequal.R

```diff
@@ -117,3 +118,28 @@ check_names_ok <- function(x, label) {
     stop("All elements in `", label, "` must be named")
   }
 }
+
+#' @export
+#' @rdname expect_setequal
+expect_contains <- function(object, expected) {
```

Comment:
I am wondering if this should just be `expect_in(needles, haystack)`?

And the error could be like 

```r
fail(paste0(
      act$lab, " isn't fully contained within ", exp$lab, ".\n",
      paste0("* Unaccounted for in `actual`: ",  values(act$val[act_miss]), "\n"),
      paste0("* Present in `expected`:   ",  values(exp$val), "\n")
    ))
```

Reply:
I will say it took me longer to understand `expect_contains()` because it doesn't map directly to `%in%` since the args are backwards

Reply:
Hmmmm, good point. But it's also going to be confusing if its the only expectation where the expected values come first. Maybe this is an argument for keeping it as `expect_contains()`?

Reply:
> the only expectation where the expected values come first.

Maybe I'd have a different use case than you? I see it as:

```r
values <- generate_output()
expect_in(values, some_set_of_possible_results)
```

Reply:
I see it used in places more like `expect_contains(dir(tempdir(), "foo.csv")`.

Reply:
Might as well include it now?